descriptions/css/properties/animation.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "animation": {
+        "__short_description": "The <strong><code>animation</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS/Shorthand_properties'>shorthand</a> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property sets an animated transition between styles."
+      }
```

Comment:
I think just "...applies a CSS animation" would be better. I think "transition" might be a confusing word, given CSS transitions.